biggraphite/drivers/cassandra.py

```diff
@@ -1485,6 +1485,11 @@ def glob_directory_names(self, glob, start_time=None, end_time=None):
         directory_names = self.__glob_names("directories", glob)
         return bg_glob.filter_from_glob(directory_names, glob)
 
+    def glob_metrics(self, glob, start_time=None, end_time=None):
+        """Return a sorted list of metrics matching this glob."""
+        return [self.get_metric(metric_name)
```

Comment:
maybe it would be worth trying to use execute_concurrent here to have all the requests be started in parallel

Reply:
Done (that changed a few things in the accessor)